examples/flutter_gallery/lib/demo/contacts_demo.dart

```diff
@@ -93,7 +93,7 @@ class ContactsDemoState extends State<ContactsDemo> {
   static final GlobalKey<ScaffoldState> _scaffoldKey = new GlobalKey<ScaffoldState>();
   final double _appBarHeight = 256.0;
 
-  AppBarBehavior _appBarBehavior;
+  AppBarBehavior _appBarBehavior = AppBarBehavior.pinned;
```

Comment:
Mostly just consistency with the Gallery home (though I'll admit I like it better on iOS too). I can switch to normal.